.ado/jobs/e2e-test.yml

```
@@ -51,11 +50,20 @@ jobs:
           workingDirectory: $(Build.SourcesDirectory)
 
       - task: CmdLine@2
-        displayName: Set up AppVerifer on ReactUWPTestApp
+        displayName: Set up AppVerifier on ReactUWPTestApp
         inputs:
           script: regedit /S $(Build.SourcesDirectory)\.ado\scripts\ReactUWPTestApp.reg
           workingDirectory: $(Build.SourcesDirectory)
-        condition: false # Must be manually enabled, since it causes a 5x perf reduction that causes test instability
+
+      - task: powershell@2
+        displayName: Install WInAppDriver 1.2.1
```

```suggestion
        displayName: Install WinAppDriver 1.2.1
```

This _should_ be already installed if we are using the standard azdo image but we may not be. See https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md